src/RawTransaction.php

```diff
@@ -1032,7 +1032,11 @@ public static function create($inputs, $outputs, $magic_byte = null, $magic_p2sh
         $tx_array['vout'] = array();
         foreach ($outputs as $address => $value) {
             if (!BitcoinLib::validate_address($address, $magic_byte, $magic_p2sh_byte)) {
-                return false;
+                throw new \Exception("Invalid address [{$address}]");
+            }
+
+            if (!is_int($value)) {
```

Comment:
I'm not entirely sure if this is smart.  

In our SDK we're using a check for a `.` or `,` instead of `is_int`, I think the main reason was because the bcmath stuff returns string, gonna go and double check this.